### DISCUSSION TITLE
1.11 or 2.0

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#517] [General]
The last major update for ToonSquid was released 3 months ago. And since then, I've been wondering on what the next update will have in store for us. Out of Pegs was a really nice addition, though there are some things that can be done in order to improve it, such as letting us edit it whenever we want without having to reset it. One thing that'd really be useful is the addition of a fixed onion skin toggle feature to help with more limited animation styles.

**Comment by happygoluckycodeeditor** [comment]
I would also like to have a Blur Effect which has keyframing and also Keyframing for warp and distort in the transform settings

**Comment by DinoAwakening** [comment]
All we need now is a rework for the fill bucket tool. If Keiwan makes it just as good as RoughAnimator’s (which shouldn’t be that hard since he already made vectors work in many kind of ways), the community will come up with more finished animations more often. Cause one of the biggest problems in ToonSquid is the way you have to manually color each drawing.
